Tools/ardupilotwaf/boards.py

```diff
@@ -17,6 +17,9 @@ def __init__(cls, name, bases, dct):
         if cls.abstract:
             return
 
+        if 'toolchain' not in cls.__dict__:
```

Comment:
Ah, cool! I have to say I don't know Python enough for this part so I just followed what you had done for name. I will correct it and do a force push